docs/guides/reference/ard-expanded-processing-extent.md

```diff
@@ -47,17 +52,17 @@ This table contains the full list of locations, including the additional locatio
   - 122-067, 123-067, 48LWP, 48LXP 
 * - Cocos (Keeling) Island
   - 128-068, 128-069, 47LKG, 47LKH, 46LHN, 46LHM 
-* - Talbot Islands, Dauan Island, Saibai Island and Boigu Island
+* - Talbot Islands, Dauan Island, Saibai Island, and Boigu Island
   - 099-066 
-* - Daru, Western Province Papua New Guinea and Northern Warrior Reefs
+* - Daru, Western Province Papua New Guinea, and Northern Warrior Reefs
   - 098-066, 54LZR
 * - Elizabeth Reef
   - 085-081, 57JVG, 57JWG 
 * - Middleton Reef
   - 085-080, 57JVH, 57JWH 
 * - Frederick Reef, Queensland
   - 088-075, 56KQB, 57KTS, 56KQA 
-* - Coral Sea – Scenes with Ocean depth less than 200m
+* - Coral Sea (scenes with Ocean depth less than 200 m)

```

Comment:
@OHehir-Aaron  Can we remove the reference to 200 m here? It was a very informal rule and not used to actually select the scenes (that was done via assessing Australia's territorial sea boundaries I think)

Reply:
Sure!